tasks/main.yml

```diff
@@ -15,6 +15,9 @@
 
 - include: setup_cluster.yml
 
+- include: mysql_users.yml
+  tags: mysql-users
+
```

Comment:
@hamza-tumturk This is great stuff man. I think we should add a `when: mariadb_mysql_users is defined` here though. Other than that it is much appreciated.